Tensile/Configs/rocblas_sgemm_nn_inc1_asm_full.yaml

```diff
@@ -210,143 +215,160 @@ BenchmarkProblems:
           - [32, 32, 1, 2]
         - PrefetchGlobalRead: [True]
         - ThreadTile:
+          - [ 1, 32 ]
```

Comment:
These files have too many thread tile combinations. This will only work if NewClient: 0 and MergeFiles: False